source/databases-collections.txt

```diff
@@ -1,13 +1,135 @@
 .. _vsce-databases-collections:
 
-==============================================
-Navigate Databases, Collections, and Documents
-==============================================
+==================
+Navigate Your Data
+==================
 
 .. default-domain:: mongodb
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+Once you connect to your deployment using |vsce|, use the left
+navigation to:
+
+- Explore your databases, collections, and documents. 
+
+- Create new databases and collections.
+
+- Drop databases and collections.
+
+.. figure:: /images/dbs-colls-docs.png
+  :figwidth: 300px
+  :alt: Image showing deployment navigation
+
+Databases and Collections
+-------------------------
+
+When you expand an active connection, |vsce| shows the databases in that
+deployment. Click a database to view the collections it contains.
+
+View Collection Documents and Schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click a collection to view its documents and schema.
+
+Documents
+``````````
+
+When you expand a collection's documents, |vsce| lists the ``_id`` of
+each document in the collection. Click an ``_id`` value to open that
+document in VS Code and view its contents.
+
+Alternatively, right-click a collection and click
+:guilabel:`View Documents` to view all of the collection's documents
+in an array.
+
+Opening collection documents provides a **read-only** view of your data.
+To modify your data using |vsce|, use a JavaScript Playground.
+
+Schema
+``````
+
+Your collection's schema defines the fields and data types within the
+collection. Due to MongoDB's flexible schema model, different documents
+in a collection may contain different fields, and data types may vary
+within a field. MongoDB can enforce
+:manual:`schema validation </core/schema-validation/>` to ensure your
+collection documents have the same shape.
```

Comment:
nice nice. The only lingering question about this for me: can you edit schemas through VSCE? I'm guessing not, but if so we might want to point folks toward an explanation! Since you can edit everything else discussed on this page (docs via javascript playground, dbs, cols), it might be useful to add a quick clarification about editing schemas anyway at the end of this section.

Reply:
Going to defer to product on this one. @mmarcon, as far as I know there is no way to edit a schema using the extension (only view). Inserting new documents will modify the schema (which is maybe behavior worth clarifying), but I'm just confirming they can't set schema validation rules in the extension?